rear_rider_device/actuators/led_strip.py

```diff
@@ -174,11 +239,12 @@ def update_elapsed_time():
     led_strip = LedStripController(pixels)
     # TODO add an Effect that selects leds in a group to apply an effect to. Effect mask?
     try:
-        effects: list[LedStripEffect] = [
-            StrobeEffect()
-        ]
-        enter_leds_effects_loop(effects=effects, fps=DEFAULT_FPS)
-    except Exception as e:
+        loop_ctx = LedsEffectsLoopContext(
+            led_strip_ctrl=led_strip,
+            frame=LedStripFrame(fps=DEFAULT_FPS))
+        loop_ctx.set_effects([StrobeEffect()])
+        enter_leds_effects_loop(loop_ctx)
+    finally:
         print('turning off')
```

Comment:
to further clarify, moving what you did to the `finally` block makes sense, but if there's an actual exception because something is borked we should catch that

Reply:
Are you saying that the code in the `finally` block may raise an Exception? Or am I misinterpreting you?

Reply:
nah, i was saying add an exception handler back in, since it was removed. i tried to say that moving the code you did was the right choice, but we still should do something when an exception occurs